examples/guided.py

```diff
@@ -237,8 +318,13 @@ def ask_user_questions():
 def perform_installation_steps():
 	print()
 	print('This is your chosen configuration:')
-	archinstall.log("-- Guided template chosen (with below config) --", level=logging.DEBUG)
-	archinstall.log(json.dumps(archinstall.arguments, indent=4, sort_keys=True, cls=archinstall.JSON), level=logging.INFO)
+	archinstall.log(
+		"-- Guided template chosen (with below config) --", level=logging.DEBUG
+	)
+	archinstall.log(
+		json.dumps(archinstall.arguments, indent=4, sort_keys=True, cls=archinstall.JSON),
+		level=logging.INFO,
+	)
```

Comment:
Disclaimer (going to be a lot of these), this following is my personal opinion:

This decreases readability. It completely mixes and chooses what to do based on the position of the moon.
Split all arguments on individual lines or don't do antyhing at all. And if you're doing it, make sure you do it to the `json.dumps()` call too damn it.